canary/models/Statistical.md

```diff
@@ -1,298 +0,0 @@
-
-# Statistical Model
```

Comment:
Why is this being deleted?

Reply:
Do we need `.md` files? It's a bit messy with all those pictures...